Update release notes for v1.6.7 (#2251)
Automattic/studio: RELEASE-NOTES.txt

Unreleased
==========

1.6.7
=====
* Fixed issues with app auto-updates #2234

> * Fixed issues when creating new sites after pulling an existing site #2230

Reviewer: ```suggestion
* Fixed creation of the new sites after pulling an existing site #2230
```

> * Fixed issues with Studio Assistant plugin and theme installation #2200, #2243

Reviewer: We can also possibly remove "issues" in there as I don't think it adds much to the meaning
